### PR TITLE
Add explaining-signet to bitcoin-magazine/bitcoin-explained

### PR DESCRIPTION
**Pull request by spraveenitpro**
This PR adds [Explaining Signet](https://www.youtube.com/watch?v=lGJaIbpf6bk) transcript to the bitcoin-magazine/bitcoin-explained directory.

### CODE REVIEW

File: bitcoin-magazine/bitcoin-explained/explaining-signet.md [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+media: https://www.youtube.com/watch?v=lGJaIbpf6bk
+tags: ["signet"]
+speakers: ["Sjors Provoost","Aaron Van Wirdum"]
+categories: ["podcast"]
+date: 2020-09-24
+---
+
+## intro
+
+Speaker 0: 00:00:07
```
Comment (adamjonas) [inline bitcoin-magazine/bitcoin-explained/explaining-signet.md:13]: These speaker names need to be replaced throughout

File: bitcoin-magazine/bitcoin-explained/explaining-signet.md [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+
+Live from Utrecht, this is the fan weirdum Sjoerds Nedo. Hello! Sjoerds, welcome.
```
Comment (adamjonas) [inline bitcoin-magazine/bitcoin-explained/explaining-signet.md:15]: The show name and speakers names need updates

File: bitcoin-magazine/bitcoin-explained/explaining-signet.md [hunk 3]
```diff
@@ -0,0 +1,527 @@
...
+
+Okay.
+
+Speaker 1: 00:01:47
+
+And now that Bitcoin is not worthless, you know, you don't want to test things on production. In general, you do not want to test things on production. So there's this additional network that is identical to Bitcoin and that's called Testnet.
+
+Speaker 0: 00:02:00
+
+Yeah, it's basically a clone of mainnet which was created I think by, I think Gavin and Driesen created this back in, what would have been 2011, 12, 13? Probably before, I don't know.
```
Comment (adamjonas) [inline bitcoin-magazine/bitcoin-explained/explaining-signet.md:115]: Gavin's name needs updating

**Review by adamjonas — CHANGES_REQUESTED** [review]
@spraveenitpro There are a number for updates, especially people's names, needed before this is ready for merge.

**Comment by kouloumos** [comment]
@spraveenitpro, Since we haven't got any response from you, I'll go ahead and close this submission. This will provide others with the chance to review it or for you to to revisit it.

We hope to see your work in the future. Thank you!

**Pull request closed without merging**